xml/System.Text/Rune.xml

```diff
@@ -594,8 +610,8 @@ This method is very similar to <xref:System.Text.Rune.DecodeFromUtf8(System.Read
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Returns <see langword="true" /> if and only if this scalar value is within the BMP ([ U+0000..U+FFFF ]) and therefore representable by a single UTF-16 code unit.</summary>
-        <value>To be added.</value>
+        <summary>Returns <see langword="true" /> if and only if this scalar value is within the BMP ([ U+0000..U+FFFF ]) and therefore representable by a single UTF-16 code unit; <see langword="false" /> otherwise.</summary>
+        <value>A boolean indicating whether this scalar value is withing the BMP or not.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if the scalar value associated is within the BMP encoding range ([ U+0000..U+FFFF ]) and therefore representable by a single UTF-16 code unit; otherwise, <see langword="false" />.</value>
```